pkg/hive/hive.go

```diff
@@ -204,7 +198,7 @@ func (s *Service) sendPeers(ctx context.Context, peer swarm.Address, peers []swa
 			return err
 		}
 
-		if !s.allowPrivateCIDRs && isPeerPublic && manet.IsPrivateAddr(addr.Underlay) {
+		if !s.allowPrivateCIDRs && manet.IsPrivateAddr(addr.Underlay) {
```

Comment:
We need to verify this does not break our private testnet-type environment. In the private-testnet, I think they dont use publicly available IPs. Also as all the nodes are in the same VPS, they can connect using the private IP.